kolibri/deployment/default/settings/base.py

```diff
@@ -54,6 +54,7 @@
     'django.contrib.sessions',
     'django.contrib.messages',
     'django.contrib.staticfiles',
+    'django_filters',
```

Comment:
This line seems to fix the broken browseable API issue. I needed to add this line after using django_filters for Lessons to view it on browseable API page.